package.json

```diff
@@ -1,62 +1,24 @@
 {
   "name": "querystring",
   "id": "querystring",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "Node's querystring module for all engines.",
-  "keywords": [ "commonjs", "query", "querystring" ],
   "author": "Irakli Gozalishvili <rfobic@gmail.com>",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/Gozala/querystring.git",
-    "web": "https://github.com/Gozala/querystring"
-  },
-  "bugs": {
-    "url": "http://github.com/Gozala/querystring/issues/"
-  },
-  "devDependencies": {
-    "test": "~0.x.0",
-    "phantomify": "~0.x.0",
-    "retape": "~0.x.0",
-    "tape": "~0.1.5"
-  },
+  "license": "MIT",
+  "repository": "Gozala/querystring",
   "engines": {
     "node": ">=0.4.x"
   },
   "scripts": {
-    "test": "npm run test-node && npm run test-browser && npm run test-tap",
-    "test-browser": "node ./node_modules/phantomify/bin/cmd.js ./test/common-index.js",
-    "test-node": "node ./test/common-index.js",
-    "test-tap": "node ./test/tap-index.js"
+    "test": "jest"
```

Comment:
This package was tested in browser engines, while if I see correctly it's no longer the case after this change

Reply:
true, like earlier browser testing isn't done since jest uses jsdom by default, if not jest then also any suggestion is welcome :)

Reply:
Was testing as setup currently in `master` broken?

Reply:
yeah, `test-browser` command was throwing error, also package `phantomify` isn't updated in last 8 years

Reply:
Yes, it appears it depends on `phantomjs` to be installed in a system, which on its own is deprecated